src/test/regress/sql/scale_factor.sql

```diff
@@ -0,0 +1,247 @@
+-- start_ignore
```

Comment:
This test is not included to scheduler and contains mistakes.

Reply:
This test was fixed and included in scheduler

Reply:
There is dedicated greenplum_schedule for gpdb-specific tests.